internal/pkg/agent/storage/storage.go

```diff
@@ -17,7 +17,23 @@ type Store interface {
 	Save(io.Reader) error
 }
 
+type Storage interface {
```

Comment:
Where is this interface used? I see references to `storage.Store`, but none for this

Reply:
right here https://github.com/elastic/elastic-agent/pull/398/files#diff-2a61fe9fd49a564217db2978adbb5374f41245e897461350c0a1d09b6471bad0R37